sql/2019/13_Ecommerce/13_02.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 13_02: Top shops
+SELECT
+  _TABLE_SUFFIX AS client,
+  app,
+  COUNT(0) AS freq,
+  total,
+  ROUND(COUNT(0) * 100 / total, 2) AS pct
+FROM
+  `httparchive.technologies.2019_07_01_*`
+JOIN
+  (SELECT _TABLE_SUFFIX, COUNT(0) AS total FROM `httparchive.summary_pages.2019_07_01_*` GROUP BY _TABLE_SUFFIX)
+USING (_TABLE_SUFFIX)
```

Comment:
Not sure I follow. This query counts the number of pages per app per client and divides it by the number of pages per client. So if a page has multiple apps, both will be counted since there would be a row for each in the technologies table.

Reply:
Ah that makes sense. Disregard this then :)